docs/api/ethereum.md

```diff
@@ -1,5 +1,645 @@
 # API / Ethereum
 
+## Bitski back-end provider
+
+A [Bitski](https://www.bitski.com/) back-end provider is applied back-end Node.js usage. You will be able to connect to ethereum blockchain and create queries and mutation via your Bitski developer wallets. To setup Bitski back-end provider it is required to first create a [development account](https://developer.bitski.com/) on Bitski.
+
+### BitskiProvider(options)
+
+A `class` providing the communication with the Ethereum blockchain through [Bitski](https://www.bitski.com/).
+
+**Arguments**
+
+| Argument | Description
+|-|-|-
+| options.assetLedgerSource | A `string` representing the URL to the compiled ERC-721 related smart contract definition file. This file is used when deploying new asset ledgers to the network.
+| options.clientId | A `string` representing the Bitski client id. You get the client ID by creating a [developer account](https://developer.bitski.com/) on Bitski.
```

Comment:
`| options.clientId | A `string` representing the Bitski client ID. You get the client ID by creating a [developer account](https://developer.bitski.com/) on Bitski.`